dids/src/main/kotlin/web5/sdk/dids/methods/dht/DhtClient.kt

```diff
@@ -86,7 +88,8 @@ internal class DhtClient(
 
     val response = runBlocking { client.get("${gateway}/${id}") }
     if (!response.status.isSuccess()) {
-      throw RuntimeException("Error getting message from DHT")
+      val err = runBlocking { response.bodyAsText() }
+      throw PkarrRecordResponseException("Error reading Pkarr Record Set of id $id. Error: $err")
```

Comment:
would love to know what id i should pass in when writing a unit test to check this exception is thrown. to do this, i need to pass in an `id` that's z-based 32 encoded bytes that when passed to diddht.tbddev.org/${id}, will return a 400, 404, or 500. so far i cannot get it to give me anything other than 200 🤔

Reply:
cc: @decentralgabe

Reply:
followed up with gabe, seems like there's no easy way for us to come up with an ID that will pass the ZBase32.decode() but fail the client call to diddht.tbddev.org/{id}.